.gitignore

```diff
@@ -1,4 +1,5 @@
 /vendor
 composer.lock
 /phpunit.xml
+/.phpunit.cache
 .phpunit.result.cache
```

Comment:
Remove `.phpunit.result.cache`.

Reply:
I think it's safer to leave this be because I already ran in the case where I accidentally almost committed this on my local Laravel 10 skeleton because I almost forgot to remove this file. Think a lot of contributors might run into the same issue.